### PR TITLE
MAINT: Stop printing full boilerplate, ``black fmriprep/cli``

### PR DESCRIPTION
**Pull request by oesteban**
This PR stops the printing of the full boilerplate in the standard output.

I have run ``black`` on the cli module and fixed a tiny bug with a formatting argument not being interpolated when reporting missing dependencies.

### CODE REVIEW

**Comment by tsalo** [comment]
Does this resolve #1349?

**Comment by effigies** [comment]
This is hard to review. If you're going to `black` things, can you make that a separate commit so that the semantic changes can be seen and evaluated separately?

I would suggest something like:

```
git rebase -i upstream/master^
# set `master` to `e`
black <files>
git commit -a -m 'STY: black'
git rebase --continue
# resolve all conflicts in favor of 1a2a299
```

**Comment by oesteban** [comment]
@effigies sorry about that, I separated it in two commits

@tsalo, yes, I made a reference to resolve that issue. Thanks pointing it out.

File: fmriprep/cli/run.py [hunk 1]
```diff
@@ -108,11 +108,13 @@ def main():
             sentry_sdk.capture_message(success_message, level='info')
 
         # Bother users with the boilerplate only iff the workflow went okay.
-        if (config.execution.output_dir / 'fmriprep' / 'logs' / 'CITATION.md').exists():
+        _boiler_file = config.execution.output_dir / "fmriprep" / "logs" / "CITATION.md"
+        if _boiler_file.exists():
```
Comment (effigies) [inline fmriprep/cli/run.py:121]: We're scoped inside a function. Private variables don't mean anything.

```suggestion
        boiler_file = config.execution.output_dir / "fmriprep" / "logs" / "CITATION.md"
        if boiler_file.exists():
```

File: fmriprep/cli/run.py [hunk 2]
```diff
@@ -108,11 +108,13 @@ def main():
...
+            _boiler_file = _boiler_file.relative_to(config.execution.output_dir)
```
Comment (effigies) [inline fmriprep/cli/run.py:122]: If somebody's using Singularity or manually constructing a Docker command, they'll be able to parse the output directory. What if we condition this on `fmriprep-docker`'s mapping to `/out`?

```suggestion
            if str(config.execution.output_dir) == "/out
                boiler_file = Path("<output_dir>") / boiler_file.relative_to(config.execution.output_dir)
```

File: fmriprep/cli/run.py [hunk 3]
```diff
@@ -108,11 +108,13 @@ def main():
...
             config.loggers.workflow.log(
-                25, 'Works derived from this fMRIPrep execution should '
-                'include the following boilerplate:\n\n%s',
-                (config.execution.output_dir / 'fmriprep' / 'logs' / 'CITATION.md').read_text()
+                25,
+                "Works derived from this fMRIPrep execution should include the "
+                f"boilerplate text found in <output_dir>/{_boiler_file}.",
```
Comment (effigies) [inline fmriprep/cli/run.py:126]:
```suggestion
                f"boilerplate text found in {boiler_file}.",
```

**Comment by oesteban** [comment]
> I'm going to assume that the `black` call was semantically neutral.

I just called `black fmriprep/cli` and then fixed a couple of pep8 errors it introduced (apparently it likes to have trailing commas after list elements and other semantically neutral stuff). So yes, semantically neutral.

**Pull request merged**